draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -294,6 +294,53 @@ This example shows a 300MB remaining quota for an application in the next 60 sec
    RateLimit: "default";r=300000000;t=60;pk=:QXBwLTk5OQ==:
 ~~~
 
+# Problem Types (#problem-types)
+
+## Quota Exceeded
+
+This section defines the "https://iana.org/assignments/http-problem-types#quota-exceeded" problem type. A server MAY use this problem type if it wants to communicate to the client that the requests sent by the client exceed one or more Quota Policies. This problem type defines the extension member "violated-policies" as an array of strings.
+
+```http
```

Comment:
If `~~~ http-message` is used instead of ```http, the message can be validated by a CI job. Not strictly necessary, but potentially helpful. See https://httpwg.org/admin/editors/tools-formats#ci and https://httpwg.org/admin/editors/style-guide#example-messages.